ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/constants/RdfConstants.java

```diff
@@ -11,6 +11,7 @@ private RdfConstants() {
 	public static final String TREE = "https://w3id.org/tree#";
 	public static final Property TREE_VIEW = createProperty(TREE, "view");
 	public static final String TREE_VIEW_DESCRIPTION = TREE + "viewDescription";
+	public static final String TREE_VIEW_DESCRIPTION_RESOURCE = TREE + "ViewDescription";
```

Comment:
Duplicate of the one above

Reply:
No, the casing is different. One is a property, the other one a class

Reply:
Toppie!